File: pgm/ABC_Java/ABC_338_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final char[] S = reader.readLine().toCharArray();
        final String Yes = "Yes";
        final String No = "No";
        if (!Character.isUpperCase (S[0])) {
            System.out.println (No);
            return;
        } else {
            for (int i = 1; i < S.length; i++) {
                if (!Character.isLowerCase (S[i])) {
                    System.out.println (No);
                    return;
                }
            }
        }
        System.out.println (Yes);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_338_A.kt
// import java.time.*
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.util.StringTokenizer

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"

fun main() {
    val S: String = readln()
    if (S == S.lowercase().replaceFirstChar { it.uppercase() }) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}